deploy/helm/sumologic/values.yaml

```diff
@@ -89,6 +107,10 @@ sumologic:
     # Source category for the Events source. Default: "{clusterName}/events"
     sourceCategory: ""
 
+  # Traces configuration
+  traces:
+    enable: false
```

Comment:
nit - for consitency with other flags to control can you change to `enabled`

Reply:
also, lets make sure we make it clear that the tracing configuration is alpha and not for general use.

Reply:
@perk-sumo 
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/drosiek-zipkin-plugin/deploy/helm/sumologic/values.yaml#L92

I will also change `enable` -> `enabled`

Reply:
fixed; I wanted to be consistent with previous PRs but I like `enabled` more :)